distribution/lib/Standard/Google_Api/0.0.0-dev/src/Google_Credential.enso

```diff
@@ -0,0 +1,31 @@
+from Standard.Base import all
+from Standard.Base.Metadata import make_single_choice, Widget
+
+polyglot java import com.google.api.gax.core.CredentialsProvider
+polyglot java import com.google.api.gax.core.FixedCredentialsProvider
+polyglot java import com.google.auth.oauth2.GoogleCredentials
+
+polyglot java import org.enso.aws.ProfileReader
+
+type Google_Credential
+
+    ## ICON key
+       Access using a json credentials file downloaded from your Google account.
+
+       Arguments:
+       - file: Path to the json credentials file.
+    From_File file:File
+
+    ## ICON key
+       Access using the defaults provided by the environment variables.
+    Default
+
+    ## PRIVATE
+    as_java : CredentialsProvider
```

Comment:
Can the `## PRIVATE` method be moved into `private` module? A module which starts with `private` keyword?

Reply:
Yes. Done